src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+
+public class Circle extends Figure implements FigureDrawer {
```

Comment:
```suggestion
public class Circle extends Figure {
```
якщо батьківський клас реалізовує інтерфейс це не потрібно повторювати в дочірніх